test/controllers/SecureMessagingControllerSpec.scala

```diff
@@ -224,41 +221,40 @@ class SecureMessagingControllerSpec extends ControllerSpecBase with TestRequests
-        controller.submitReply(clientId, conversationId)(postRequest).futureValue
+        val result = controller.submitReply(clientId, conversationId)(postRequest)
 
-        verify(connector).submitReply(any[String], any[String], any[Map[String, Seq[String]]])(any[HeaderCarrier])
+        status(result) mustBe OK
+
+        val expectedUrl = Some(routes.SecureMessagingController.displayInbox.url)
+        verify(partial_wrapper).apply(any(), any(), any(), eqTo(expectedUrl))(any(), any())
       }
 
-      "returns a successful Future of None" that {
-        "wraps the returned partial in the partial_wrapper page" in {
-          secureMessagingFeatureAction.enableSecureMessagingFeature()
-          when(connector.submitReply(any[String], any[String], any[Map[String, Seq[String]]])(any[HeaderCarrier]))
-            .thenReturn(Future.successful(Some(ConversationPartial("<html></html>"))))
+      "call secure message connector" in {
+        secureMessagingFeatureAction.enableSecureMessagingFeature()
+        when(connector.submitReply(any[String], any[String], any[Map[String, Seq[String]]])(any[HeaderCarrier]))
+          .thenReturn(Future.successful(None))
 
-          val postRequest = fakeRequest.withFormUrlEncodedBody("messageReply" -> "BlaBla").withCSRFToken
-          val result = controller.submitReply(clientId, conversationId)(postRequest)
+        val postRequest = fakeRequest.withFormUrlEncodedBody("messageReply" -> "BlaBla").withCSRFToken
+        controller.submitReply(clientId, conversationId)(postRequest).futureValue
 
-          status(result) mustBe OK
-        }
+        verify(connector).submitReply(any[String], any[String], any[Map[String, Seq[String]]])(any[HeaderCarrier])
       }
 
-      "returns a successful Future of Some(ConversationPartial)" that {
-        "wraps the returned partial in the partial_wrapper page" in {
-          secureMessagingFeatureAction.enableSecureMessagingFeature()
-          when(connector.submitReply(any[String], any[String], any[Map[String, Seq[String]]])(any[HeaderCarrier]))
-            .thenReturn(Future.successful(Some(ConversationPartial("<html></html>"))))
+      "returns a successful Future of None and wraps the returned partial in the partial_wrapper page" in {
+        secureMessagingFeatureAction.enableSecureMessagingFeature()
+        when(connector.submitReply(any[String], any[String], any[Map[String, Seq[String]]])(any[HeaderCarrier]))
+          .thenReturn(Future.successful(Some(ConversationPartial("<html></html>"))))
 
-          val postRequest = fakeRequest.withFormUrlEncodedBody("messageReply" -> "BlaBla").withCSRFToken
-          val result = controller.submitReply(clientId, conversationId)(postRequest)
+        val postRequest = fakeRequest.withFormUrlEncodedBody("messageReply" -> "BlaBla").withCSRFToken
+        val result = controller.submitReply(clientId, conversationId)(postRequest)
 
-          status(result) mustBe OK
-        }
+        status(result) mustBe OK
```

Comment:
I am not sure, but based on the test name it seems to me there might be an assertion missing. To check if partial was wrapped.